src/Compiler/Driver/CompilerOptions.fs

```diff
@@ -1388,6 +1389,11 @@ let testFlag tcConfigB =
             | "ContinueAfterParseFailure" -> tcConfigB.continueAfterParseFailure <- true
             | "ParallelOff" -> tcConfigB.concurrentBuild <- false
             | "ParallelCheckingWithSignatureFilesOn" -> tcConfigB.parallelCheckingWithSignatureFiles <- true
+            | "PartiallyParallelOptimization" ->
```

Comment:
`Partially` here refers to the fact that I have plans to increase the level of parallelisation by using the TypedTree dependency graph. Perhaps a different prefix would be more suitable, eg. `PhaseBased`.

Reply:
I'd suggest being specific here, just like in the flag above: what exactly is being parallelized :)

Reply:
I changed it to `ParallelOptimization`. Not really sure what details I could include that would reasonably fit in a commandline argument. I'm open to suggestions though.